internal/backend/user.go

```diff
@@ -52,6 +55,24 @@ func newUser(
 		return nil, err
 	}
 
+	// Create recovery mailbox if it does not exist
+	recoveryMBox, err := db.WriteResult(ctx, database, func(ctx context.Context, tx *ent.Tx) (*ent.Mailbox, error) {
+		mboxFlags := imap.NewFlagSet(imap.FlagSeen, imap.FlagFlagged, imap.FlagDeleted)
```

Comment:
do we need to support \Seen and \Flagged? Why not just \Deleted?

Reply:
We should be able to change the flagged status of the message. I see no reason why we can't support this.

Reply:
If these flags not supported, does this mean we should not allow this to be modified then?

Reply:
good question, i don't even know what gluon does if a client tries to interact with flags we don't list as supported 🤔 i guess in theory we support any flag, even those we don't announce as supported.

Reply:
I don't think we have anything that's checking  for that. TBD in a separate ticket/MR